content/en/blog/2020/addon-rework/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Reworking our Addon Integrations
+description: A new way to manage installation of telemetry addons.
+publishdate: 2020-06-10
```

Comment:
@howardjohn Do you want to hold off on posting this until June 10? If not, change this date, and we can merge it as soon as you're ready.

Reply:
No, I just picked an arbitrary date in the future. I'll switch it to today